sdcm/db_stats.py

```diff
@@ -357,7 +357,12 @@ def _init_stats(self):
     def get_scylla_versions(self):
         versions = {}
         try:
-            versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
+            node = self.db_cluster.nodes[0]
+            if node.is_rhel_like():
+                version_cmd = 'rpm -qa |grep scylla'
+            else:
+                version_cmd = "dpkg -l |grep scylla|awk '{print $2 \"-\" $3}'"
```

Comment:
'dpkg' command looks fine, but it seems that 'apt list --installed' or 'dpkg-query -l' are generally preferred.

Reply:
We can switch it to dpkg-query.
The result of apt list --installed contained many unused info, we have to filter them out